modules/shutup/src/main/Dictionary.scala

```diff
@@ -27,8 +27,8 @@ ass?(hole|fag)
 autist(ic|)
 aus?c?hwitz
 bastard?
-be[ea]+ch
-bit?ch
+be[ea]+t?ch
+biy?t?ch
```

Comment:
Is `biych`/`biytch` really something ppl write? i'd maybe expect `bytch`. though probably won't have false positives. but maybe then just do `b[iy]+t?ch`?

Reply:
I was also fairly surprised to see it but according to the internal discussion it is sometimes used